neutralino.js/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "echo \"Error: no test specified\" && exit 0",
```

Comment:
nit: should not say `Error` and exit with code `0`.

Reply:
This is getting removed in the next PR right after this, so can we merge it knowing this